File: S1/1/A_1_2.c
// Болібрук Влад 
#include <stdio.h>
#include <stdlib.h>
int main() {
		
	int x = -1;
	int y = 1;
	
	float result = (abs(x)-abs(y))/(1+abs(x*y));
	
	printf("task is A_1_2\n");
	printf("result is %f\n",result);
	
	return 0;	
}

File: S1/3/A_1_2.java
// Болібрук Влад 

public class A_1_2 {
	public static void main(String[] args) {
		int x = -1;
		int y = 1;

		float result = (Math.abs(x) - Math.abs(y)) / (1 + Math.abs(x * y));

		System.out.println("task is A_1_2");
		System.out.printf("result is %f\n", result);
	}
}
